examples/asserts/rules-entity-relation.yml

```diff
@@ -19,23 +19,23 @@ entities:
       - query: |
           group by (asserts_env, asserts_site, k8s_dst_namespace, k8s_dst_owner_name) (beyla_network_flow_bytes_total{k8s_dst_owner_type=~"Pod|Deployment|Node|DaemonSet|ReplicaSet|StatefulSet"})
```

Comment:
Can we delete this file entirely?

This needs to be a core part of Asserts, and not something we should ask users configure on their end.

Reply:
Actually this is not part of the documentation but some in-repository examples (mainly for internal usage)